reference_guide/api_changes_list_2023.md

```diff
@@ -128,6 +128,50 @@ Fragment builder functions from `ExternalSystemRunConfigurationUtil` file moved
 `org.jetbrains.kotlin.idea.actions.JavaToKotlinAction.Companion` class renamed to `org.jetbrains.kotlin.idea.actions.JavaToKotlinAction.Handler`
 : In order to not load additional code eagerly on action instantiation.
 
+### Markdown Plugin 2023.3
+
+`org.intellij.plugins.markdown.editor.images` package removed
+: Use `org.intellij.plugins.markdown.images` from `intellij.markdown.images` module.
+
+`org.intellij.plugins.markdown.lang.psi.MarkdownPsiElementFactory.createHtmlBlockWithImage(Project, MarkdownImageData)` method removed
+: Use `org.intellij.plugins.markdown.images.editor.ImagePsiElementFactory.createHtmlBlockWithImage` instead.
+
+`org.intellij.plugins.markdown.lang.psi.MarkdownPsiElementFactory.createHtmlImageTag(Project, MarkdownImageData)` method removed
+: Use `org.intellij.plugins.markdown.images.editor.ImagePsiElementFactory.createHtmlImageTag` instead.
+
+`org.intellij.plugins.markdown.lang.psi.MarkdownPsiElementFactory.createImage(Project, String, String, String)` method removed
+: Use `org.intellij.plugins.markdown.images.editor.ImagePsiElementFactory.createImage`.
+
+`action.org.intellij.plugins.markdown.ui.actions.styling.InsertImageAction.insert.popup.text` removed from resource bundle `messages.MarkdownBundle`
```

Comment:
please fix to 
`(<identifier>) <<<property>>> removed from resource bundle (<identifier>)`

Reply:
Sorry, but I don't think that I understand what you mean here. What should be the first `(<identifier>)`? Can you give an example?

Reply:
this entry misses word `property` to be a valid pattern